grammar/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "ebnf-helper"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+#ebnf = "0.1.4"
+ebnf = { git = "https://github.com/torao/ebnf" }
```

Comment:
Is this unspecified dev version still needed? It'd be preferable to pin the standard to fixed versions